esp/src/src-react/util/metricGraph.ts

```diff
@@ -627,16 +640,28 @@ export class MetricGraphWidget extends SVGZoomWidget {
         return this;
     }
 
+    enter(domNode, element) {
+        super.enter(domNode, element);
+    }
+
     update(domNode, element) {
         super.update(domNode, element);
     }
 
+    exit(domNode, element) {
+        super.exit(domNode, element);
+    }
+
     async renderSVG(svg: string): Promise<void> {
         return new Promise<void>((resolve, reject) => {
             this._selection = {};
             const startPos = svg.indexOf("<g id=");
             const endPos = svg.indexOf("</svg>");
-            this._renderElement.html(svg.substring(startPos, endPos));
+            this._renderElement.html(svg.substring(startPos, endPos)
+                .split("\"black\"").join("var(--colorNeutralForeground1)")
+                .split("\"white\"").join("var(--colorNeutralBackground1)")
+                .split("\"whitesmoke\"").join("var(--colorNeutralBackground2)")
```

Comment:
Minor complaint, but split / join seems weird to me. I'd think chaining .replaceAll(...)s would be more obvious what was happening